src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,58 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+
+export interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: ((selectedTabId: string) => void),
+};
+
+export const Tabs: React.FC<Props> = React.memo(
+  ({
+    tabs,
+    selectedTabId,
+    onTabSelected,
+  }) => {
+    const isSelectedTab = tabs.find((tab) => selectedTabId === tab.id)
+     || tabs[0];
+
+    return (
+      <>
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map((tab) => (
+              <li
+                key={tab.id}
+                className={classNames({
+                  'is-active': tab.id === isSelectedTab.id,
+                })}
+                data-cy="Tab"
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (tab.id !== isSelectedTab.id) {
+                      onTabSelected(tab.id);
+                    }
```

Comment:
Here is the problem with the test. It requires you to pass the tab as an argument.

Reply:
I am not sure, so you are saying that instead of passing tab.id I should pass the whole object tab, somehow? is this correct?

Reply:
Yup you found it 👍